<launch>
  <arg name="mav_name" default="auk"/>
  <arg name="namespace" default="$(arg mav_name)" />

  <group ns="$(arg namespace)" >
    <node name="esdf_server" pkg="voxblox_ros" type="esdf_server" output="screen" args="--alsologtostderr" clear_params="true">
      <remap from="pointcloud" to="stereo_dense_reconstruction/pcl"/>
      <param name="tsdf_voxel_size" value="0.05" />
      <param name="tsdf_voxels_per_side" value="16" />
      <param name="voxel_carving_enabled" value="true" />
      <param name="color_mode" value="color" />
      <param name="use_tf_transforms" value="true" />
      <param name="update_mesh_every_n_sec" value="0.0" />
      <param name="verbose" value="false" />
    </node>
  </group>
</launch>
